src/test/scala/chiselTests/util/SRAMSpec.scala

```diff
@@ -228,25 +228,6 @@ class SRAMSpec extends ChiselFlatSpec {
     val sv = emitSystemVerilog(new Top)
   }
 
-  it should "be possible to access SRAM description information" in {
```

Comment:
I don't think the needs of accessing the value of "SRAM description information" at elaboration time. actually we just choose nest inner instance's OM to outer module and access them at circt.
See our use case: https://github.com/chipsalliance/t1/pull/791/commits/ec76744ae76c3f2ceebd0a71142fae1d66ddb4f1

Reply:
We need the ability to access just the `Path`, that is our specific use case. The `Path` only isn't there right now because firtool currently doesn't have the ability to materialize `Path` Properties in leaf modules, but that's a bug that needs to be fixed.

Reply:
If there can be a way to deal with classes and select specific fields that would be great, but AFAIK that does not exist, maybe @mikeurbach will have more info.

Reply:
Maybe we can add an additional property for `Path` in SRAM? It that possible to workaround?